en/lessons/ecto/querying.md

```diff
@@ -164,12 +163,14 @@ SELECT m0."title" FROM "movies" AS m0 []
 The good thing about macros is that they work very well with pipes:
 
 ```elixir
-iex> query = Movie |> where([m], m.id < 2) |> select([m], {m.title})
-
-iex> Repo.all(query)
+iex> > Movie \ 
```

Comment:
looks like theres an extra `>` after `iex>` here

Reply:
sorry was the copy and paste on my terminal..

Reply:
```suggestion
iex> Movie \ 
```